packages/palette/README.md

```diff
@@ -0,0 +1,71 @@
+# Palette
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/palette.svg)
+
+This package contains the colors for the following palettes:
+
+- **UI**: MongoDB-branded, general-use UI palette.
+- _Chart (coming soon)_: Color palette for accessible charting colors.
```

Comment:
What do you envision the API for these will be?

We'll have `import { uiColors, chartColors, syntaxColors } from '@leafygreen-ui/palette';` ?

Seems redundant to have colors appended to every value. 

I'm not sure there's anything better.

Could do `import palette from '.'` and then have `palette.ui` etc. But then you will always include all colors. Which may be ok as it's small?

Thoughts?

Reply:
I thought a bunch about this. I _think_ this is the best solution to avoid everyone's imports looking like:

```js
import { ui as uiColors } from '@leafygreen-ui/palette';
```

I think it's important to keep the three palettes separate, and usually you will only be importing `uiColors`. `syntaxColors` will be used mostly internally in leafygreen-ui, and chart colors is pretty niche in where it should be used as well.

Reply:
Makes sense to me!